### PR TITLE
B3 GSID and MAPS fixes

### PR DESCRIPTION
**Pull request by biyeun**
- fixes issue with CDN and maps causing errors on prod
- fixes the multi location filter that the GSID reports are using.

@czue cc @benrudolph

### CODE REVIEW

**Comment by benrudolph** [comment]
lgtm will let tests go

**Comment by biyeun** [comment]
@benrudolph  tests passed

File: corehq/apps/reports/standard/maps.py [hunk 1]
```diff
@@ -251,11 +252,17 @@ def show_in_navigation(cls, domain=None, project=None, user=None):
 class ElasticSearchMapReport(GetParamsMixin, GenericTabularReport, GenericMapReport):
 
     report_template_path = "reports/async/maps.html"
-    report_partial_path = "reports/async/partials/maps.html"
+    report_partial_path = "reports/bootstrap3/partials/base_maps.html"
```
Comment (snopoke) [inline corehq/apps/reports/standard/maps.py]: Why the change from 'maps.html' to 'base_maps.html'?

File: corehq/apps/style/templates/style/bootstrap3/base.html [hunk 2]
```diff
@@ -473,6 +482,32 @@
...
+        <script src="{% static 'reports/javascripts/maps_utils.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
+
+        {% if request.use_maps_async %}
+        {% compress js %}
+        <script src="{% static 'reports/javascripts/async_maps.js' %}"></script>
+        <script src="{% static 'reports/javascripts/maps_utils.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
```
Comment (snopoke) [inline corehq/apps/style/templates/style/bootstrap3/base.html]: Moving these here seems to have impacted the timing for when they get loaded: http://manage.dimagi.com/default.asp?222977

It looks like the partial page is evaluated before these are loaded. Locally the JS files are loaded first so I'm not able to reproduce.

Comment (biyeun) [inline corehq/apps/style/templates/style/bootstrap3/base.html]: @snopoke I don't believe that this PR is the cause of those issues. I actually think it's the fix. Look at the timestamp of the issue vs when this PR got merged in. I believe this isn't in master yet? The ticket seems like it's talking about the CDN issues that this PR addresses.

Comment (biyeun) [inline corehq/apps/style/templates/style/bootstrap3/base.html]: I tested the reports yesterday (before implementing this fix) and they were showing those symptoms since the CDN was causing those JS files to load after the inline JS had been executed.

Comment (snopoke) [inline corehq/apps/style/templates/style/bootstrap3/base.html]: Appologies - I had assumed this was already on prod. I'll kick the ticket back to interrupt to verify its fixed after deploy.

File: corehq/apps/reports/standard/maps.py (hunk 1 above)
Comment (biyeun) [inline corehq/apps/reports/standard/maps.py]: once I moved the js files to base rather than the async template, there was no difference between `async/partials/maps.html` and `maps_base.html`

Comment (snopoke) [inline corehq/apps/reports/standard/maps.py]: can we remove the `maps.html` file then?

Comment (biyeun) [inline corehq/apps/reports/standard/maps.py]: it's used by other b2 views that are migrated outside of this PR. once those are migrated this can be deleted

Comment (snopoke) [inline corehq/apps/reports/standard/maps.py]: 👍thanks for all the hard work on B3 stuff!
